### DISCUSSION TITLE
Does v3 not compatable with v2 syntax anymore?

### DISCUSSION THREAD

**Discussion by AoGao-Kedoka** [jeffreylanters/unity-tweens#39] [Q&A] [answered]
It seems that I can't use v2 syntax in v3 anymore. Is that correct?

Upgrading from existing v2 project will cause errors since package name is changed. In my case in order to use v2, `"nl.elraccoone.tweens": "git+https://github.com/jeffreylanters/unity-tweens#v2.1.0"` is the way to install & prevent update.

**Comment by jeffreylanters** [comment, marked as answer]
Hi. That's correct, the release of major version 3 introduces a more modern way to create Tweens. But feel free to keep using version 2.x, I will always remain available with LTS!
